chemistry/gromacs/gromacstop.py

```diff
@@ -747,6 +747,7 @@ def update_typelist_from(ptypes, types):
                             break
         update_typelist_from(params.dihedral_types, self.dihedral_types)
         update_typelist_from(params.improper_periodic_types, self.dihedral_types)
+        self.update_dihedral_exclusions()
```

Comment:
Let's move this below where we copy the Ryckaerts-Bellemans torsions, since those are also Dihedrals that will impact exclusions.